interop/xds/client/client.go

```diff
@@ -280,12 +378,11 @@ func makeOneRPC(c testpb.TestServiceClient, cfg *rpcConfig) (*peer.Peer, *rpcInf
 	return &p, &info, err
 }
 
-func sendRPCs(clients []testpb.TestServiceClient, cfgs []*rpcConfig, ticker *time.Ticker) {
-	var i int
+func sendRPCs(clients []testpb.TestServiceClient, ticker *time.Ticker) {
 	for range ticker.C {
-		go func(i int) {
-			// Get and increment request ID, and save a list of watchers that
-			// are interested in this RPC.
+		for i := 0; i < len(clients); i++ {
```

Comment:
This is incorrect. We don't want to send one RPC on each client when the ticker fires.

Reply:
Made this change (ticker fires now in accordance with qps, and per tick it iterates through clients).

Reply:
I think you didn't make the change requested.  IIUC he asked for the ticker to fire at the same rate, and to only send RPCs on one channel each tick.

Reply:
Made this change.